dockerenforcer/docker_helper.py

```diff
@@ -163,3 +163,53 @@ def kill_container(self, container):
             logger.error("Communication error when stopping container {0}: {1}".format(container.cid, e))
         except Exception as e:
             logger.error("Unexpected error when stopping container {0}: {1}".format(container.cid, e))
+
+    def get_recent_detections_with_all_violated_rules(self, jurek, judge):
```

Comment:
these methods should not be part of `docker_helper.py`, as they have nothing to do with requests through docker api